ci.startup.sh

```diff
@@ -27,12 +27,12 @@ if [[ -z ${JAHIA_LICENSE} ]]; then
 fi
 
 echo "$(date +'%d %B %Y - %k:%M') == Cluster enabled: ${JAHIA_CLUSTER_ENABLED} =="
-if [[ "${JAHIA_CLUSTER_ENABLED}" == "true" ]]; then
+if [[ "${JAHIA_CLUSTER_ENABLED}" == "true" ||  ${JAHIA_CLUSTER_ENABLED} == true ]]; then
```

Comment:
I'm not sure why that same condition is present twice.

If you're unsure about the format of the variable, it's probably best to address the problem upstream of this check